demo/site/server.js

```diff
@@ -5,40 +5,50 @@ const next = require("next");
 const fs = require("fs");
 
 const dev = process.env.NODE_ENV !== "production";
-const port = process.env.APP_PORT ?? 3000;
+const hostname = "localhost";
+const port = process.env.APP_PORT ?? 3333;
```

Comment:
Why change the port?

Reply:
this was for testing purposes (removed now)